docs/make.jl

```diff
@@ -32,7 +32,7 @@ makedocs(;
     warnonly = true,
     checkdocs=:all,
     format=DocumenterVitepress.MarkdownVitepress(
-        repo = "github.com/LuxDL/DocumenterVitepress.jl", # this must be the full URL!
+        repo = "github.com/tpoisot/DocumenterVitepress.jl", # this must be the full URL!
```

Comment:
```suggestion
        repo = "github.com/LuxDL/DocumenterVitepress.jl", # this must be the full URL!
```
you need to change this one back, before we merge or run the PR.

Reply:
✅